src-tauri/src/lib.rs

```diff
@@ -277,12 +277,15 @@ pub fn launch_northstar(
             return Err(anyhow!("Not all checks were met").to_string());
         }
 
-        // Require Origin to be running to launch Northstar
-        let origin_is_running = check_origin_running();
-        if !origin_is_running {
-            return Err(
-                anyhow!("Origin not running, start Origin before launching Northstar").to_string(),
-            );
+        // Require Origin to be running to launch Northstar except for Steam
+        if !matches!(game_install.install_type, InstallType::STEAM)
+        {
+            let origin_is_running = check_origin_running();
+            if !origin_is_running {
+                return Err(
+                    anyhow!("Origin not running, start Origin before launching Northstar").to_string(),
+                );
+            }
         }
```

Comment:
Should this be moved to the code below that runs `NorthstarLauncher.exe`?

Reply:
Marking as _resolved_ as no longer relevant ^^